### PR TITLE
Migrate to Chakra UI

### LINKED ISSUE #97: Accessiblity needs to be fixed.
There are a ton of accessibility issues on the site. Like properly following h1.2.3.4.5.6 order and alt-text and aria-labels not even existing.

### PR DESCRIPTION
**Pull request by anirudhb**
Closes #97, mostly

Be warned this code is extremely hacky so be prepared to gouge your eyes out lol

### CODE REVIEW

File: frontend/pages/teams/[id].tsx [hunk 1]
```diff
@@ -94,7 +109,7 @@ function InviteModal({
     if (query == "") {
       setUsers([]);
     } else {
-      fetchApi(`/users/search?excludeSelf=true&q=${query}`).then((res) => {
+      fetchApi(`/users/search?excluseSelf=true&q=${query}`).then((res) => {
```
Comment (jasonappah) [inline frontend/pages/teams/[id].tsx]:
```suggestion
      fetchApi(`/users/search?excludeSelf=true&q=${query}`).then((res) => {
```
guessing this was accidental

**Review by jasonappah — CHANGES_REQUESTED** [review]
Everything else LGTM

**Review by jasonappah — APPROVED** [review]

**Review by sarthaktexas — APPROVED** [review]

**Review by cjdenio — CHANGES_REQUESTED** [review]
Found some visual issues:
- the app component is too tall and doesn't show as a grid on the team page
- and the app component doesn't look great in dark mode
- also the "apps", "users", and "expenses" labels on the team dashboard are illegible in dark mode

Screen Shot 2021-05-04 at 8 29 45 PM
Screen Shot 2021-05-04 at 8 29 08 PM

**Review by cjdenio — APPROVED** [review]
lgtm!

**Pull request merged**